Update hardware-and-software-requirements-for-biztalk-server-2020.md (#1006)
MicrosoftDocs/biztalk-docs: biztalk/install-and-config-guides/hardware-and-software-requirements-for-biztalk-server-2020.md

description: Describes the software prerequisites and supported version lists in order to install BizTalk Server 2020.
ms.custom: biztalk-2020,devx-track-javaee-websphere
ms.service: biztalk-server

> ms.date: 01/13/2025

Reviewer: ```suggestion
ms.date: 02/13/2025
```

When applying a cumulative update, some of the required software versions may change. To see the software versions that are updated, read the KB article for the cumulative update. For a list of cumulative updates, go to [Service Pack and cumulative update list for BizTalk Server](https://support.microsoft.com/topic/service-pack-and-cumulative-update-list-for-biztalk-server-108e5e94-4558-8b57-d5fb-45984506d56f).

- **Microsoft Windows**: Required. Supported versions include:
  - Windows Server 2022 (with Desktop Experience) - requires minimum BizTalk 2020 [CU6](https://support.microsoft.com/help/5043408)
  - Windows Server 2019 (with Desktop Experience)
  - Windows Server 2016 (with Desktop Experience)
  - Windows 10

>   - Windows 11

Reviewer 2: for the sake of consistency:
- Since Windows Server versions are listed in desc order, please list Windows 11 above Windows 10
- Add the same note as Win2022/SQL 2022 for Windows 11 as well:  - requires minimum BizTalk 2020 [CU6](https://support.microsoft.com/help/5043408)

Author: done

- **WinSCP**: Required to use the SFTP adapter.

>   BizTalk requires certain WinSCP version depending on which BizTalk cumulative update that is installed. For WinSCP version details, see [SFTP adapter documentation](../core/sftp-adapter.md).

Reviewer: ```suggestion
  BizTalk requires a specific WinSCP version based on the installed BizTalk cumulative update. For WinSCP version details, see [SFTP adapter documentation](../core/sftp-adapter.md).
```

  Supported versions:

  - WinSCP version [5.15.4](https://winscp.net/download/WinSCP-5.15.4-Automation.zip) . Download automation library at [WinSCP](http://winscp.net) and extract to BizTalk installation folder.
  - BizTalk Server 2020 with CU1 or CU2 - WinSCP version [5.17.6](https://winscp.net/download/WinSCP-5.17.6-Automation.zip)
  - BizTalk Server 2020 with CU3 or CU4 - WinSCP version [5.19.2](https://winscp.net/download/WinSCP-5.19.2-Automation.zip)
  - BizTalk Server 2020 with CU5 - WinSCP version [6.1.2](https://winscp.net/download/WinSCP-6.1.2-Automation.zip)

>   - BizTalk Server 2020 with CU6 - WinSCP version [6.3.5](https://winscp.net/download/WinSCP-6.3.5-Automation.zip)

Reviewer 2: Suggestion: instead of listing the WinSCP versions in 2 different pages, I think we can redirect to the SFTP adapter page from here

For supported versions, please see [SFTP Adapter Prerequisites](../core/sftp-adapter#prerequisites)

Author: Done. Referring to SFTP-Adapter documentation in one place
